core/src/main/java/org/mapfish/print/http/HttpRequestCache.java

```diff
@@ -0,0 +1,178 @@
+package org.mapfish.print.http;
+
+
+import jsr166y.ForkJoinPool;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.client.AbstractClientHttpResponse;
+import org.springframework.http.client.ClientHttpRequest;
+import org.springframework.http.client.ClientHttpResponse;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+
+/**
+ * 
+ * Cached ClientHttpRequestFactory.
+ * 
+ * @author Niels Charlier
```

Comment:
We no longer use the @author tag, see: https://github.com/mapfish/mapfish-print/pull/474